### DISCUSSION TITLE
Sending mapped value in the URL

### DISCUSSION THREAD

**Discussion by vmanju123** [grafana/grafana#38396] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/61594112/86603922-b01ba480-bf72-11ea-9308-3382d177c107.png)

**What happened**:
In above screenshot, Server IP column in elastic data base contains 0.0.0.0, we are mapping it to * using value mappings
There is a URL enabled on application column which is using {__cell_x} variable to pass context of server IP as below

/d/${__cell_4}-monitor/${__cell_4}-monitor?&from=${__from}&to=${__to}&var-clientIp=${__cell_5}&var-serverIp=${__cell_7}&var-serverPort=${__cell_8}&var-clientPort=${__cell_6}

the URL always picks up 0.0.0.0 but not * as mapped value 

**What you expected to happen**:
if a value is mapped, url {__cell_x} variable should pick mapped value and not whats in data

**How to reproduce it (as minimally and precisely as possible)**:
map a value from any column, use {__cell_x} in URL and check that mapped value is not picked up by {__cell_x} variable

**Anything else we need to know?**: nothing

**Environment**:
- Grafana version: 6.5.2
- Data source type & version: elasticsearch
- OS Grafana is installed on: ubuntu 18.4
- User OS & Browser: windows chrome browser
- Grafana plugins: table plugin
- Others:

**Comment by vmanju123** [comment]
also tried {__cell_x:raw} and {__cell_x:text} but result is same, it always picks whats in the data but not mapped value

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
